contracts/wrapper/WrapAndUnWrapUniV3.sol

```diff
@@ -0,0 +1,792 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0 <0.9.0;
+
+import "@openzeppelin/contracts/token/ERC721/IERC721.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "@openzeppelin/contracts/utils/math/SafeMath.sol";
```

Comment:
Remove `SafeMath.sol` and all it's usage in the code because it's no longer needed in solidity 0.8.0